### PR TITLE
Launching middleware required in testing with docker

### PR DESCRIPTION
**Pull request by yoh1496**
There are middleware needed to run test of `personium-engine`

- ElasticSearch
- ActiveMQ
- memcached
- nginx
- Tomcat

I write docker-compose to launch them excepting Tomcat.
( You need launch Tomcat locally and deploy `personium-core` there )

How about this?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,14 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 
+## Testing
+
+1. Place `personium-ex-mailsender` jar file and `Ext_MailSender.properties` file in `/personium/personium-engine/`
+1. Launch prequire software with `docker-compose up`
```
Comment (dixonsiu) [inline README.md]:
```suggestion
1. Launch required software with `docker-compose up`
```

File: README.md [hunk 2]
```diff
@@ -25,6 +25,14 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
...
+1. Launch tomcat9 locally
```
Comment (dixonsiu) [inline README.md]:
```suggestion
1. Launch tomcat9 locally  
`systemctl start tomcat`
```

Comment (dixonsiu) [reply README.md]: It is more friendly to provide a sample command. Please use the correct tomcat service name when you reflect on the change.

File: README.md [hunk 3]
```diff
@@ -25,6 +25,14 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
...
+1. Deploy `personium-core` tomcat locally with option `io.personium.core.pathBasedCellUrl.enabled=true`
```
Comment (dixonsiu) [inline README.md]: How to deploy? What are the procedures/commands?

File: README.md [hunk 4]
```diff
@@ -25,6 +25,14 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
...
+1. Execute tests with maven.
```
Comment (dixonsiu) [inline README.md:39]: Can you provide the maven command?

**Review by dixonsiu — CHANGES_REQUESTED** [review]
Please check my suggestion.

Good job! Thank you for making personium-engine better.

**Review by shimono — APPROVED** [review]
Thanks a lot.

File: README.md (hunk 2 above)
Comment (yoh1496) [reply README.md]: I'll write launch and deploy command

**Review by yoh1496 — COMMENTED** [review]
I updated. Could you check this?

**Review by dixonsiu — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -25,6 +25,23 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
...
+For example, with below comand. ( Set `$TOMCAT_DIR` environment value to path of directory which Tomcat9 installed, and place `personium-core.war` on `$TOMCAT_DIR/webapps/personium-core.war`. )
+
+    ```
+    JAVA_OPTS="-Djava.util.logging.config.file=$TOMCAT_DIR/conf/logging.properties -Djava.util.logging.manager=org.apache.juli.ClassLoaderLogManager -Djdk.tls.ephemeralDHKeySize=2048 -Djava.protocol.handler.pkgs=org.apache.catalina.webresources -Dorg.apache.catalina.security.SecurityListener.UMASK=0027 -Dcatalina.base=$TOMCAT_DIR -Dcatalina.home=$TOMCAT_DIR -Djava.io.tmpdir=$TOMCAT_DIR/temp -Dio.personium.configurationFile=$PWD/src/test/resources/personium-core-unit-config.properties" catalina.sh run
+    ```
+
+1. Execute tests with maven.
+
+    ```
+    mvn -f pom.xml
```
Comment (tochi-y) [inline README.md]:
```suggestion
    mvn -f pom.xml test
```

I think if no specifying phase, package, install and deploy phases are executed.

**Review by tochi-y — CHANGES_REQUESTED** [review]

**Review by tochi-y — APPROVED** [review]

**Pull request merged**